sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharp.java

```diff
@@ -36,4 +36,8 @@ public CSharp(Settings settings) {
     return settings.getStringArray(CSharpPlugin.FILE_SUFFIXES_KEY);
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Why have you added this? Seems totally useless to override a method just to call super on it...